src/main/kotlin/no/nav/familie/tilbake/dokumentbestilling/felles/BrevmottagerUtil.kt

```diff
@@ -9,6 +9,9 @@ import no.nav.familie.tilbake.behandling.domain.Verge as DomainVerge
 object BrevmottagerUtil {
 
     fun getannenMottagersNavn(brevmetadata: Brevmetadata): String? {
```

Comment:
```suggestion
    fun getAnnenMottagersNavn(brevmetadata: Brevmetadata): String? {
```

Eksisterende kode men -